lib/java_buildpack/container/tomee/tomee_instance.rb

```diff
@@ -34,6 +44,21 @@ def tomcat_7_compatible
         @version < TOMEE_7
       end
 
+      private
+
+      def web_inf_lib
+        if ear?
+          # copy additional libraries to tomee lib folder
+          tomcat_lib
```

Comment:
Tried once more @violetagg, it looks like for database driver defined via resources.xml file the jars in ear/lib is not resolving but work when placed in tomee/lib. For shared functionality in different wars however ear/lib location works.